src/components/CardForm/CardForm.tsx

```diff
@@ -0,0 +1,71 @@
+import { FormEvent } from 'react';
+import CardNumberSection from '../../components/CardNumberSection/CardNumberSection';
+import CardExpirationSection from '../../components/CardExpirationSection/CardExpirationSection';
+import CardCompanySection from '../../components/CardCompanySection/CardCompanySection';
+import CvcSection from '../../components/CvcSection/CvcSection';
+import PasswordSection from '../../components/PasswordSection/PasswordSection';
+import Button from '../../components/Button/Button';
+import { useNavigate } from 'react-router';
+import { useStack } from '../../hooks/useStack';
+import { useStepFlow } from '../../hooks/useStepFlow';
+import { STEPS } from '../../constants';
+import styles from '../../../src/pages/HomePage/HomePage.module.css';
+import useValues from '../../hooks/useValues';
+import { useFormValid } from '../../hooks/useFormValid';
+
+type CardFormProps = ReturnType<typeof useValues>['cardForm'];
+
+export function CardForm(props: CardFormProps) {
+  const { Stack, setStep } = useStack<(typeof STEPS)[number]>('카드번호');
+
+  const { cardNumbers, company, expiration, cvc, password } = props;
+
+  useStepFlow({ setStep, cvc: cvc.cvc, expiration: expiration.expiration, company: company.company, cardNumbers: cardNumbers.cardNumbers });
```

Comment:
```suggestion
  useStepFlow({ setStep, cvc: cvc.cvcState, expiration: expiration.expirationState, company: company.companyState, cardNumbers: cardNumbers.cardNumbersState });
```
cvc 객체 안에 cvc 속성이 있는게 약간 어색하게 느껴지는 것 같아요..! 

`cvc` 객체 안에 `cvcState` 같이 상태라는걸 알리는 네이밍은 어떨까요?!

Reply:
좋은 지적입니다! 고쳐보려고 했으나.. 쉽지 않네요 대공사가 될 거 같아서 패스하겠습니다 ㅠㅠ